packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -122,6 +125,12 @@ export enum TelemetryTiggerFrom {
   Other = "Other",
 }
 
+export enum WatchVideoFrom {
+  WatchVideoBtn = "WatchVideoBtn",
+  PlayBtn = "PlayBtn",
+  WatchOnBrowserBtn = "WatchOnBrowserBtn",
```

Comment:
how about specified the "From" in event, instead of trigger-from. To align with the report. https://msit.powerbi.com/groups/d71619a2-07f7-4363-96c0-6fc3fac03214/reports/7e682b6c-3f38-414e-812d-7816a881ebfc/ReportSection7c0cdeafd9db0040c78b